docs/machine-learning/how-to-guides/explain-machine-learning-model-permutation-feature-importance-ml-net.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "How-To: Explain ML.NET Machine Learning Model Using Permutation Feature Importance"
+description: Understand the feature importance of models with Permutation Feature Importance in ML.NET
+ms.date: 04/19/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use Permutation Feature Importance in ML.NET to determine the feature importance of my models so that I can understand how my machine learning models make decisions and which features contribute to their performance.
+---
+
+# How-To: Explain ML.NET Machine Learning Model Using Permutation Feature Importance
+
+Learn to explain ML.NET machine learning models by understanding the contribution features have to predictions using Permutation Feature Importance (PFI).
```

Comment:
Is this first time we've mentioned explainability. We might need to define it here.

Reply:
#resolved